foxpuppet/windows/browser/notifications.py

```diff
@@ -0,0 +1,114 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from abc import ABCMeta
+
+from selenium.webdriver.common.by import By
+
+
+class BaseNotification(object):
```

Comment:
Whatever works best. As said earlier nothing is set in stone yet. This was just a thinking... I find it's looking superfluous.